var/spack/repos/builtin/packages/plasma/package.py

```diff
@@ -21,17 +21,17 @@ class Plasma(MakefilePackage):
     problems, and singular value problems."""
 
     homepage = "https://bitbucket.org/icl/plasma/"
-    url      = "https://bitbucket.org/icl/plasma/downloads/plasma-17.1.tar.gz"
-    hg       = "https://luszczek@bitbucket.org/icl/plasma"
+    url = "https://bitbucket.org/icl/plasma/downloads/plasma18.9.0.tar.gz"
+    hg = "https://luszczek@bitbucket.org/icl/plasma"
 
     version("develop", hg=hg)
-    version("17.1", "64b410b76023a41b3f07a5f0dca554e1")
```

Comment:
typically we don't remove old versions

Reply:
Acknowledged, but the old version had no configuration system and the new one has CMake. The Spack package class has to either extend MakefilePackage or CMakePackage as far as I understand.

Reply:
Yes, but you can override the Cmake-specific methods and implement the make flow in e.g. the install method. See for example the nest package

Reply:
I added the old version back that uses Makefile. nest package was useful - thanks.